target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -87,6 +87,7 @@ ramips_setup_interfaces()
 	comfast,cf-wr758ac-v1|\
 	comfast,cf-wr758ac-v2|\
 	tplink,tl-wr902ac-v3|\
+	tplink,tl-wr902ac-v4|\
```

Comment:
If your device only has one ethernet port, you can move it to the first case to use `ucidef_set_interface_lan "eth0"`. I don't know why a single port switch is needed here. Looking forward to your feedback if there are some strange behaviors.

Reply:
See new commit. I connected phone via wifi and I was able to ping it from computer connected by eth. So I think it works.